### ISSUE TITLE
Write files to internal storage 

### ISSUE THREAD

**Issue by DmitryM9** [LiquidPlayer/LiquidCore#104]
You can somehow save files from Micro service to internal storage. It is necessary to permanent storage, even when you restart the app and privacy files. Is there such a possibility?

**Comment by ericwlange** [comment]
Hi Yes.  Please see the [documentation on the file system](https://github.com/LiquidPlayer/LiquidCore/wiki/LiquidCore-File-System).  For example, you can read and write from `/home/local` and this will be persisted across usages of the microservice.

**Comment by DmitryM9** [comment]
> All content in this directory will persist so long as a micro service in this namespace is installed on the host

Can you describe in detail the behavior of the library in this place?

**Comment by ericwlange** [comment]
Yes, I realize this is complicated wording and I will have to figure out how to make it simpler.

Let's say you have a MicroService with a URI of `https://myservice.io/service1/liquid.bundle`:  And also assume you have two different apps: APP_A and APP_B which both use that service.

When you write to `/home/local` from either APP_A or APP_B, what is written there will only be accessible to the app which wrote it.  So if APP_A writes `myfile.txt`, that file will be visible only to APP_A's instance of the service and not APP_B.  That file will persist until one of two things happens:
1. The user uninstalls APP_A, or
2. You explicitly call [`MicroService.uninstall()`](https://liquidplayer.github.io/LiquidCoreAndroid/0.6.1/org/liquidplayer/service/MicroService.html#uninstall-Context-java.net.URI-) on the URI from APP_A.
Otherwise, the file will persist indefinitely.

When you use `/home/public/data`, it works similarly to `/home/local`, except it is accessible to _both_ APP_A and APP_B.  Files here will persist unless you call `MicroService.uninstall()`.  Uninstalling the apps will not delete these files.  The documentation suggests that if the user uninstalls both APP_A and APP_B, then these files will also get deleted.  This is absolutely my intention, but currently that doesn't happen.

Note that most use cases do not require `/home/public/data`.  You would only want to use that in the case where you have multiple Android apps that want to share the same microservice.  Honestly there aren't many rational use cases for this, so I don't recommend using it.  I may ultimately remove it from LiquidCore because it could be potentially misused, but for now consider it experimental.  Also note that `/home/public/data` does not exist on iOS at all because Apple does not even make this possible.

Bottom line: stick with `/home/local`.  It will do what you expect: it persists the files until either you explicitly uninstall them through the microservice, or the user deletes the app.

**Comment by DmitryM9** [comment]
Fine. Now the behavior is clear
